core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -120,7 +122,7 @@ public static void setupContent() throws FileNotFoundException {
             .withExposedPorts(80)
             .withExtraHost("somehost", "192.168.1.10")
             .withCommand("/bin/sh", "-c", "while true; do cat /etc/hosts | nc -l -p 80; done");
-
+    
 //    @Test
```

Comment:
the code wasn't commented in this PR.
Although +1 for removing this added space